app/components/elements/button/base.js

```diff
@@ -0,0 +1 @@
+export default props => <button {...props} />
```

Comment:
This feels like overkill.. I don't think we need a base for every component -- especially if a component is 1 line.

Reply:
I thought about this, but the reason I stuck to it was because i think it makes it easier for css /html people. As long as they can always look at base to see what html gets rendered it should make it easier to style I think.